pytorch_lightning/utilities/memory.py

```diff
@@ -25,6 +24,17 @@
 from pytorch_lightning.utilities.apply_func import apply_to_collection
 
 
+class ByteCounter:
+    def __init__(self) -> None:
+        self.nbytes: int = 0
+
+    def write(self, data: bytes) -> None:
+        self.nbytes += len(data)
+
+    def flush(self) -> None:
```

Comment:
```suggestion
    def write(self, data: bytes) -> None:
        self.nbytes += len(data)

    def flush(self) -> None:
```
Would like to some docstrings in these methods